src/guide/bash.md

```diff
@@ -53,7 +53,7 @@ is phased out and `iroha_python` is the official scripting interface, we
 might use the excellent Python libraries for creating a useful TUI. For
 now, one must make do with what one has.
 
-## 1. Iroha 2 Client Setup
+## 1. Set up Iroha 2 Client
```

Comment:
```suggestion
## 1. How to set up an Iroha 2 Client
```
Alternatively, the continuous verb forms in headings are common, especially in software documentation. 

I think we need to have a **brief** discussion about this.

Reply:
in this case we have steps we expect the user to follow so the imperative works better than "how to"